### ISSUE TITLE
Mi notebook 14 horizon hackintosh

### ISSUE THREAD

**Issue by aadil-sengupta** [uttufy/uttufy#1]
Hey Uttusharma, I saw all the work you have done on the hackintosh for MI notebook
if you dont mind me asking,
I tried you OC 0.67 version efi which works perfectly fine except, sleep isnt working for me on it
but on version 0.5 sleep is working but the trackpad isnt. I saw all your documentarions but i still wasnt able to do anything bout the trackpad in version 0.5. i wanted to use your pre-compiled dsdts and ssdts but the links no longer work ;(, i tried making my own ssdts , first i tried ssdtime which didnt work well, so i was hoping if you could help me make my own ssdt or send me a link to your precompiled ssdt's .

Thanks :)[
[dsdt.aml.zip](https://github.com/uttusharma/uttusharma/files/6665171/dsdt.aml.zip)

](url)

**Comment by uttufy** [comment]
I am working on upgrading to OC 0.7.1 in that I will be addressing your issue. 
Also you won't be need to dump and compile ssdt in the newer upgrade.

**Comment by uttufy** [comment]
Check out the latest release 
[https://github.com/uttusharma/Xiaomi-Notebook14-Hackintosh](https://github.com/uttusharma/Xiaomi-Notebook14-Hackintosh)

**Comment by aadil-sengupta** [comment]
> Check out the latest release 
> [https://github.com/uttusharma/Xiaomi-Notebook14-Hackintosh](https://github.com/uttusharma/Xiaomi-Notebook14-Hackintosh)

Hey, tested your latest efi, unfortunately sleep is still not working :/

I'll try this again soon

**Comment by uttufy** [comment]
> > Check out the latest release
> > https://github.com/uttusharma/Xiaomi-Notebook14-Hackintosh
> 
> Hey, tested your latest efi, unfortunately sleep is still not working :/
> 
> I'll try this again soon

Make sure to clear NvRAM and rebuild kext cache.
Which processor you have? is it i7 or i5?

**Comment by aadil-sengupta** [comment]
> > > Check out the latest release
> > > https://github.com/uttusharma/Xiaomi-Notebook14-Hackintosh
> > 
> > Hey, tested your latest efi, unfortunately sleep is still not working :/
> > 
> > I'll try this again soon
> 
> Make sure to clear NvRAM and rebuild kext cache.
> Which processor you have? is it i7 or i5? 
> 

I tried clearing the NVRam but when I do so, I am unable to boot an have to replace the EFI, I5 Horizon

**Comment by uttufy** [comment]
> able to boot an have to replace the EFI, I5 Horizon

So any issue appear after clear NVRAM is linked ACPI.
Try this EFI also [https://github.com/itsabhishekolkha/Mi-notebook-14-Hackintosh](https://github.com/itsabhishekolkha/Mi-notebook-14-Hackintosh)

**Comment by aadil-sengupta** [comment]
> > able to boot an have to replace the EFI, I5 Horizon
> 
> So any issue appear after clear NVRAM is linked ACPI.
> Try this EFI also [https://github.com/itsabhishekolkha/Mi-notebook-14-Hackintosh](https://github.com/itsabhishekolkha/Mi-notebook-14-Hackintosh)

Sure, I'll give it a try, as of rn am kinda busy

**Comment by aadil-sengupta** [comment]
> > able to boot an have to replace the EFI, I5 Horizon
> 
> So any issue appear after clear NVRAM is linked ACPI.
> Try this EFI also [https://github.com/itsabhishekolkha/Mi-notebook-14-Hackintosh](https://github.com/itsabhishekolkha/Mi-notebook-14-Hackintosh)

Didn't work, On possible reasonnon my mind is, in order to get imessages working I installed Catalina and updated to big sur. another user, used the efi and sleep was working for him no problem.the only difference between the 2 systems was the changed serial and the fact tht I updated from Catalina,

Just wanted to know if this is a possible reason for why sleep isn't working for me?

Thanks :)

**Comment by Harp-pixel** [comment]
Can I use  it on Mi notebook non horizon edition (i5 10th gen , mx250gpu)
will your efi works with bigSur 11.6.3? (as I want to install that)

**Comment by uttufy** [comment]
> Can I use it on Mi notebook non horizon edition (i5 10th gen , mx250gpu) will your efi works with bigSur 11.6.3? (as I want to install that)

Yes, I have tested this EFI on the same machine config. but with mac Monterey.

**Comment by Harp-pixel** [comment]
> > Can I use it on Mi notebook non horizon edition (i5 10th gen , mx250gpu) will your efi works with bigSur 11.6.3? (as I want to install that)
> 
> Yes, I have tested this EFI on the same machine config. but with mac Monterey.

can you Please specify the exact version of the Monterey that you have tested . or I can go with the latest one?
and I have to download macOS Monterey from Olarilla or from other site ? THANKS for your Help!

**Comment by uttufy** [comment]
![image](https://user-images.githubusercontent.com/16814257/154197742-837d2557-b499-4e86-b0d6-0731698f81de.png)


Available in repo README.md
